### DISCUSSION TITLE
Bank Reconciliation functionality desired?

### DISCUSSION THREAD

**Discussion by robert-daniels** [frappe/books#407] [Ideas]
I'm clicking around on the application and not seeing the ability to perform a bank reconciliation based on increases / decreases to cash. As an accountant, I use this functionality often and look for such when running tax returns. Is this a desired functionality if I can spin something up? Currently in school for information systems, and it seems like a project i'd like to try.

**Comment by robert-daniels** [comment]
What I was somewhat thinking of
_
[BankRec mockup_61922.pdf](https://github.com/frappe/books/files/8936389/BankRec.mockup_61922.pdf)
_

**Reply by ralekeo** [reply]
Hi Roberts,

This would not be a bad idea

**Comment by promexio** [comment]
Perhaps using the import functionality - depends on the format of your bank statement: re-format this to match it with the standard import function.

**Comment by TwoTwenty** [comment]
I desire this feature to.
